### PR TITLE
Move economic thermal costs input files into "series" directory

### PR DESCRIPTION
**Pull request by guilpier-code**

### CODE REVIEW

File: src/libs/antares/study/parts/thermal/cluster_list.cpp [hunk 1]
```diff
@@ -477,15 +477,29 @@ bool ThermalClusterList::savePreproToFolder(const AnyString& folder) const
...
+    each([&](const Data::ThermalCluster& c) {
+        assert(c.parentArea and "cluster: invalid parent area");
+        buffer.clear() << folder << SEP << c.parentArea->id << SEP << c.id();
+        ret = c.ecoInput.saveToFolder(buffer) && ret;
+    });
+    return ret;
+
+
+
+}
```
Comment (flomnes) [inline src/libs/antares/study/parts/thermal/cluster_list.cpp]:
```suggestion
    return ret;
}
```

File: src/libs/antares/study/parts/thermal/cluster_list.cpp [hunk 2]
```diff
@@ -518,20 +532,35 @@ bool ThermalClusterList::loadPreproFromFolder(Study& study,
...
+        auto& c = *(it->second);
+
+        assert(c.parentArea and "cluster: invalid parent area");
+        buffer.clear() << folder << SEP << c.parentArea->id << SEP << c.id();
+
+        bool result = c.ecoInput.loadFromFolder(study, buffer);
+        c.ComputeCostTimeSeries();
+
+        ret = result && ret;
+    }
```
Comment (flomnes) [inline src/libs/antares/study/parts/thermal/cluster_list.cpp]:
```suggestion
    for (auto& [name, c] : *this)
    {
        assert(c->parentArea and "cluster: invalid parent area");
        buffer.clear() << folder << SEP << c->parentArea->id << SEP << c->id();

        bool result = c->ecoInput.loadFromFolder(study, buffer);
        c->ComputeCostTimeSeries();

        ret = result && ret;
    }
```

**Review by flomnes — CHANGES_REQUESTED** [review]

File: src/libs/antares/study/parts/thermal/cluster_list.cpp (hunk 1 above)
Comment (guilpier-code) [reply src/libs/antares/study/parts/thermal/cluster_list.cpp]: Done

File: src/libs/antares/study/parts/thermal/cluster_list.cpp (hunk 2 above)
Comment (guilpier-code) [reply src/libs/antares/study/parts/thermal/cluster_list.cpp]: Done

**Review by flomnes — APPROVED** [review]

**Pull request merged**